server/wizard/steps/step_greeting.go

```diff
@@ -0,0 +1,34 @@
+package steps
+
+import (
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow/steps"
+)
+
+const (
+	stepNameGreeting = "greeting"
+
+	stepTitleGreeting = ""
+
+	stepDescriptionGreeting = `:wave: Welcome to Zoom for Mattermost! I'll walk you through the process here, so you can begin using the integration. Feel free to read the [documentation](https://mattermost.gitbook.io/plugin-zoom) if you'd like.
+
+Are you able to set up the integration with a Zoom admin account?`
+)
+
+func GreetingStep() steps.Step {
+	return steps.NewCustomStepBuilder(stepNameGreeting, stepTitleGreeting, stepDescriptionGreeting).
+		WithButton(steps.Button{
+			Name:  "Continue",
+			Style: steps.Primary,
+			OnClick: func(userID string) int {
+				return 0
```

Comment:
Just for my knowledge, what does it mean returning 0 or 999?

Reply:
@larkox @hanzei I was mainly following the conventions used in https://github.com/mattermost/mattermost-plugin-github/pull/527. I'm not entirely sure what each value means